cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -193,6 +196,24 @@ public void execute() {
             return;
         }
 
+        String[] cliArgs = new String[0];
+        if (!argList.isEmpty()) {
+            if (!argList.get(0).equals("--")) {
+                this.projectPath = Paths.get(argList.get(0));
+                if (argList.size() > 1 && !argList.get(1).equals("--")) {
+                    CommandUtil.printError(this.errStream,
+                            "unmatched command argument found: " + argList.get(1), testCmd, false);
+                    CommandUtil.exitError(this.exitWhenFinish);
+                    return;
+                }
+                if (argList.size() > 2 && argList.get(1).equals("--")) {
+                    cliArgs = argList.subList(2, argList.size()).toArray(new String[0]);
+                }
+            } else if (argList.size() > 1 && argList.get(0).equals("--")) {
```

Comment:
Shall we define a constant for `--` (e.g. `ARGS_SEPARATOR`) and reuse it in test and run commands?